### DISCUSSION TITLE
Order IMX-8 chip set.

### DISCUSSION THREAD

**Discussion by Strange21** [nihalpasham/rustBoot#63] [General]
Need to order the `NXP IMX8` series evaluation board. 

- [ ] Look for a list of vendors, board availability and estimated lead-time.

**Comment by nihalpasham** [comment]
Can you add a link to the list of boards (or options)?

**Comment by Strange21** [comment]
Following is the list of the chips
https://www.digikey.in/en/product-highlight/n/nxp-semi/imx-8-series-applications-processors

**Comment by nihalpasham** [comment]
So, we'll need a `NXP I.MX8M-nano` board - https://www.digikey.in/en/product-highlight/n/nxp-semi/imx-8-series-applications-processors#imx8nano

Here's an example of an i.mx8m-nano board
- `variscite i.mx8m-starter kit` - https://www.variscite.com/product/evaluation-kits/var-som-mx8m-nano-evaluation-kits/

**Comment by nihalpasham** [comment]
@Strange21 - can you find out which (i.mx8m-nano) evaluation-kits or boards are available?

**Comment by Strange21** [comment]
I came across following 2 evaluation kit. we can select any one from it. 
[nxp-8mnanod4evk-eval-kit](https://www.mouser.in/new/nxp-semiconductors/nxp-8mnanod4evk-eval-kit/)
[nxp-imx-8m-nano-ultralite-eval-kit](https://www.mouser.in/new/nxp-semiconductors/nxp-imx-8m-nano-ultralite-eval-kit/)

As Mouser is already our registered vendor therefore we can prefer them.
[nxp-8mnanod4evk-eval-kit](https://www.mouser.in/new/nxp-semiconductors/nxp-8mnanod4evk-eval-kit/) is readily available for dispatch which might take 2-3 weeks to get delivered.

**Comment by nihalpasham** [comment]
lets put in an order for the first one - [nxp-8mnanod4evk-eval-kit](https://www.mouser.in/new/nxp-semiconductors/nxp-8mnanod4evk-eval-kit/)

**Comment by nihalpasham** [comment]
@Strange21 - any luck placing the order

**Comment by Strange21** [comment]
Raised a purchase request. It is costing about 40000/- INR.

**Comment by nihalpasham** [comment]
any updates on this?
